### DISCUSSION TITLE
Update taxonomy and term layouts

### DISCUSSION THREAD

**Discussion by davefass** [nunocoracao/blowfish#278] [Ideas]
I would like to suggest `taxonomy.html` and `term.html` layouts be updated to match the styling and layout of `list.html`, along with sections added to `params.toml` to support.

It would be great to have the additional support for the background, hero, card view, etc.

Thanks for all the work on this theme...it looks great!

**Comment by nunocoracao** [comment]
Wow this is an interesting one... need time to work on this but love the idea. Thanks @davefass

**Comment by nunocoracao** [comment]
@davefass created this https://github.com/nunocoracao/blowfish/issues/282
and will further break it down as this is a complex one
